### PR TITLE
Chapter 2 corrections carlos

### PR DESCRIPTION
**Pull request by mkju19**
# What was implemented
Carlos corrections

### CODE REVIEW

File: report/sections/Preliminaries/preliminaries.tex [hunk 1]
```diff
@@ -107,13 +107,15 @@ \section*{Usability Testing}
 
 Where effectiveness regards how well the users accomplished their goal, efficiency regards the resources expended in relation to the completeness of the goals, and lastly, satisfaction regards the comfort of the system to the users.
 To test usability, one may perform a usability test. This is a study of the interaction that users have with a product. It is done to identify usability problems in a system, and results in a list of usability problems and knowledge about how easily a user is able to interact with the system.
-Before the test, the interviewers have to ensure that a test plan, tasks, and questions are ready, and that the equipment works.
+Before the test, the interviewers have to ensure that a test plan is ready.
+They also need to prepare tasks and questions for the test subjects, and ensure that the equipment works.
```
Comment (TheRumle) [inline report/sections/Preliminaries/preliminaries.tex:111]: What is the equipment? Maybe say necessary equipment is available and functioning? My next question is then what equipment is needed.

File: report/sections/Preliminaries/preliminaries.tex [hunk 2]
```diff
@@ -107,13 +107,15 @@ \section*{Usability Testing}
...
 During the test, representative users should interact with the design while solving tasks and thinking out loud. The interviewer collects data during this process. It is important that the interviewer interferes as little as possible by only providing help if the user is stuck.
-Afterwards, the interviewers analyze the data and create a list of problems from it.
-\cite{deb7}
+\cite{deb7} We use this method along with the system usability scale to perform a usability test on our system.
```
Comment (TheRumle) [inline report/sections/Preliminaries/preliminaries.tex:113]:
```suggestion
\cite{deb7} We use this method along with SUS  to perform a usability test on our system.
```

File: report/sections/Preliminaries/preliminaries.tex [hunk 3]
```diff
@@ -138,7 +140,7 @@ \subsection*{System Usability Scale} \label{SUSScore}
 	\item For even items: summarize the total score, then subtract the total score from 25
 	\item Add up the total scores and multiply that total by 2.5.
 \end{enumerate}
-Note that the score is not a percentage, but simply a score. If a system achieves a SUS score above 68 it means that it the UI is above average. If it is below 68, it is below average. \cite{adobeSUS}
+If the system achieves a SUS score above 68, it means that the UI is above average. Otherwise, if it is below 68, it is below average\cite{adobeSUS}.
 We use SUS in order to get an overall rating of our system.
 
 \section*{BenchmarkDotnet}
```
Comment (TheRumle) [inline report/sections/Preliminaries/preliminaries.tex:146]:
```suggestion
\section{BenchmarkDotnet}
```

File: report/sections/Preliminaries/preliminaries.tex (hunk 1 above)
Comment (mkju19) [inline report/sections/Preliminaries/preliminaries.tex:111]:
```suggestion
They also need to prepare tasks and questions for the test subjects, and ensure that the necessary equipment works.
```

**Pull request merged**